docs/backends/googlecloudmonitoring.mdx

```diff
@@ -39,6 +58,19 @@ Odigos currently supports Standard (non-Autopilot) GKE clusters. Exporting into
   ✅ Logs
 </Accordion>
 
+- **GCP_TIMEOUT** `string` : Timeout. The request timeout in seconds for the Google Cloud Monitoring API.
+  - This field is optional
+  - Example: `10s`
+- **GCP_PROJECT_ID** `string` : Project ID. The project ID for the Google Cloud Monitoring API.
```

Comment:
Is it intended for non-GKE users only? or can GKE users also set a specific project id and it will work with the built in authentication?

Reply:
It is possible for GKE users to specify a project ID, for example some users run their workloads in different projects then send all traces to a central observability project. it actually doesn't have anything to do with auth surprisingly, and the application credentials used will still need access to that different project ID